core/src/main/scala/io/projectglow/transformers/pipe/Piper.scala

```diff
@@ -60,10 +60,13 @@ private[projectglow] object Piper extends GlowLogging {
       outputformatter: OutputFormatter,
       cmd: Seq[String],
       env: Map[String, String],
-      df: DataFrame): DataFrame = {
-
+      df: DataFrame,
+      quarantineLocation: Option[String] = None): DataFrame = {
```

Comment:
Is this a path or a table? quarantineLocation name implies path, code does saveAsTable() which would be more accessible. Perhaps find a way to document / indicate better this is a table rather than a path.